### ISSUE TITLE
关于loss的权重

### ISSUE THREAD

**Issue by Echo-jyt** [TMElyralab/MuseTalk#246]
`loss = 2.0*loss_lip + loss_latents # add some weight to balance the loss`
作者您好，我看懂这里image的loss部分加了一个权重为2。请问这个2是怎么确定的，我想达到更好的图像生成结果，最优值有做实验确定吗？
